src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -308,11 +308,31 @@ export class RunParametersAction extends TooltipFromLabelAction {
 		const editor = this._notebookService.findNotebookEditor(context);
 		// Set defaultParameters to the parameter values in parameter cell
 		let defaultParameters = new Map<string, string>();
+		let emptyParameterCell: boolean;
 		editor.cells.forEach(cell => {
 			if (cell.isParameter) {
+				// Check if cell is empty or contains just whitespace
+				for (let x of cell.source) {
+					emptyParameterCell = /^\s*$/.test(x);
+				}
+				if (!cell.source || emptyParameterCell) {
+					// If there is no parameters in the cell indicate to user to add them
+					this.notificationService.notify({
+						severity: Severity.Info,
+						message: localize('noParametersInCell', "This notebook cannot run with parameters until there are parameters added to the parameter cell. [Learn more](https://docs.microsoft.com/sql/azure-data-studio/notebooks/notebooks-parameterization)."),
+					});
+				}
 				for (let parameter of cell.source) {
-					let param = parameter.split('=', 2);
-					defaultParameters.set(param[0].trim(), param[1].trim());
+					if (parameter.includes('=')) {
+						// For syntax that includes multiple parameters on one line (ex. x=1;y=2)
+						let multiParams = parameter.split(';');
```

Comment:
Given what we've discussed I think we should just remove this logic and tell users they need to have each parameter on its own line.